submissions/Lu-sure/post-oop/index.js

```diff
@@ -0,0 +1,154 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+  // ======== OBJECTS DEFINITIONS ========
+  class Inhabitant {
+    constructor(species, name, age, gender, limbs, saying) {
+      this.species = species;
+      this.name = name;
+      this.age = age;
+      this.gender = gender;
+      this.limbs = limbs;
+      this.saying = saying;
+    }
+
+    getName() {
+      return this.name;
+    }
+
+    getLimbs() {
+      return `I have ${this.limbs} limbs`
+    }
+
+    introduce() {
+      return `I'm ${this.age} years old ${this.species}, ${this.gender}. ${this.getLimbs()}. "${this.saying}". `
+    }
+  }
+
+  class Human extends Inhabitant {
+    constructor(name, lastName, age, gender, saying) {
+      super('human', name, age, gender, [2, 2], saying);
+      this.lastName = lastName;
+    }
+
+    getName() {
+      return `${super.getName()} ${this.lastName}`;
+    }
+
+    getLimbs() {
+      return `I have ${this.limbs[0]} hands and ${this.limbs[1]} legs`
```

Comment:
Not a really obvious interpretation.
Imagine we introduce birds. What index will represent wings? Should a developer keep it just in mind or browse the code to find it out?
Explicitness is good for maintainability.